functions/_fzf_search_directory.fish

```diff
@@ -7,6 +7,12 @@ function _fzf_search_directory --description "Search the current directory. Repl
     # unescape token because it's already quoted so backslashes will mess up the path
     set unescaped_exp_token (string unescape -- $expanded_token)
 
+    # Hide ./ prefix when fd version >= 8.3.0
+    # https://github.com/sharkdp/fd/releases/tag/v8.3.0
```

Comment:
Could you move this up to line 3?
And could you add a todo here that we'll remove this check in a few months and `--strip-cwd-prefix` is included in fd's documentation (it's not currently there) and bump up the required fd version to 8.3.0?